CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/StoreGraph.java

```diff
@@ -123,20 +125,20 @@
     private NativeValue oldValue = new NativeValue();
     private NativeValue newValue = new NativeValue();
     private GraphEdit graphEdit;
-    private int layerMaskSelectedAttributeId = -1;
-    private int vertexLayerMaskAttribureId = -1;
-    private int transactionFilterBitmaskAttrId = -1;
-    private int vertexLayerVisibilityAttributeId = -1;
-    private int transactionLayerVisibilityAttributeId = -1;
-    private int cameraAttributeId = -1;
-    private int vertexSelectedAttributeId = -1;
-    private int transactionSelectedAttributeId = -1;
-    private int vertexXAttributeId = -1;
-    private int vertexYAttributeId = -1;
-    private int vertexZAttributeId = -1;
-    private int transactionXAttributeId = -1;
-    private int transactionYAttributeId = -1;
-    private int transactionZAttributeId = -1;
+    private int layerMaskSelectedAttributeId = Graph.NOT_FOUND;
+    private int vertexLayerMaskAttribureId = Graph.NOT_FOUND;
+    private int transactionFilterBitmaskAttrId = Graph.NOT_FOUND;
+    private int vertexLayerVisibilityAttributeId = Graph.NOT_FOUND;
+    private int transactionLayerVisibilityAttributeId = Graph.NOT_FOUND;
+    private int cameraAttributeId = Graph.NOT_FOUND;
+    private int vertexSelectedAttributeId = Graph.NOT_FOUND;
+    private int transactionSelectedAttributeId = Graph.NOT_FOUND;
+    private int vertexXAttributeId = Graph.NOT_FOUND;
+    private int vertexYAttributeId = Graph.NOT_FOUND;
+    private int vertexZAttributeId = Graph.NOT_FOUND;
+    private int transactionXAttributeId = Graph.NOT_FOUND;
+    private int transactionYAttributeId = Graph.NOT_FOUND;
+    private int transactionZAttributeId = Graph.NOT_FOUND;
```

Comment:
@cygnus-x-1 @algol60 I asked @aldebaran30701 to change the layer view mask attributes to `Graph.NOT_FOUND` instead of `-1`. I wanted to ask either of you if you have any issues with this change for the camera, selection and x,y,z?

@cygnus-x-1 I would have expected you would have picked this up ages ago in your mass refactor/cleanup PRs and made them `Graph.NOT_FOUND` so a little curios/converned if there is something I should know  - is there anything special with `-1`?

I get very nervous/paranoid changing `StoreGraph`.

Reply:
@arcturus2 I was the one who added the `cameraAttributeId` and alike into `StoreGraph`
The purpose was to capture the attribute id of the camera and other commonly changed values. When it is one of the commonly changed atributes - the graph does not trigger a recalculation of layers. The recalculation of layers currently loops all layers, for every element on the graph. So this was implemented to restrict updates.

Reply:
I think I suggested this change when they were added...